.circleci/config.yml

```
@@ -82,14 +82,14 @@ jobs:
             tasksExists=$(aws --output text ecs list-task-definitions --family-prefix ${awsFamily})
             echo "existsTaskDefinition? --${tasksExists}--"
             if [ ! -z "$tasksExists" ]; then
-              isTaskRunning=$(aws --output text ecs list-tasks --cluster ordino-public --family ${awsFamily})
+              isTaskRunning=$(aws --output text ecs list-tasks --cluster caleydo --family ${awsFamily})
```

```suggestion
              isTaskRunning=$(aws --output text ecs list-tasks --cluster ordino-public --family ${awsFamily})
```

This needs to be reverted, because the productive system is on a different cluster.

Production: `ordino-public` (master branch)
Development: `caleydo` (develop branch)